Peer dependency issues

Trying to install this for the first time in my env.

npm install @assistant-ui/react @assistant-ui/react-ui -- installs fine.

```
    "@assistant-ui/react": "^0.3.5",
    "@assistant-ui/react-ui": "^0.0.18",
```

When adding: 

`npm install @assistant-ui/react-ai-sdk `

Get:

```
npm error While resolving
npm error Found: @assistant-ui/react@0.3.5
npm error node_modules/@assistant-ui/react
npm error   @assistant-ui/react@"^0.3.5" from the root project
npm error
npm error Could not resolve dependency:
npm error peer @assistant-ui/react@"^0.7.56" from @assistant-ui/react-ai-sdk@0.7.13
npm error node_modules/@assistant-ui/react-ai-sdk
npm error   @assistant-ui/react-ai-sdk@"*" from the root project
npm error
npm error Fix the upstream dependency conflict, or retry
npm error this command with --force or --legacy-peer-deps
npm error to accept an incorrect (and potentially broken) dependency resolution.
```

Installing with --legacy-peer-deps results in:

✘ [ERROR] No matching export in "node_modules/@assistant-ui/react/dist/index.mjs" for import "useThreadContext"

--- a/packages/react-ui/package.json
+++ b/packages/react-ui/package.json
@@ -1,7 +1,7 @@
 {
   "name": "@assistant-ui/react-ui",
   "description": "Pre-styled React components for @assistant-ui/react",
-  "version": "0.0.7",
+  "version": "0.1.0",
   "license": "MIT",
   "exports": {
     ".": {

